packages/docs/docs/communications/organizing-communications.md

```diff
@@ -0,0 +1,480 @@
+
+Taking our example from earlier, we could search for a thread that uses `partOf` and sort the results with the below search.
+
+**Example: **
+
+```ts
+await medplum.searchResources('Communication', {
+  id: 'example-id',
+  _revinclude: 'Communication:part-of',
+  _sort: 'sent',
+});
+```
+
+This will return the communications sorted in ascending order. You can also sort by descending order and provide more than one argument to sort by in a comma separated list. The below example sorts the thread by descending time `sent` and then by `sender`.
+
+**Example: **
+
+```ts
+await medplum.searchResources('Communication', {
+  id: 'example-id',
+  _revinclude: 'Communication:part-of',
+  _sort: '-sent,sender',
+});
+```
+
+## Codesystems
+
+It is recommended to use LOINC or SNOMED codes to classify `Communication.category` and `Communication.topic`, but it is also possible to use custom coding internal to your practice.
+
+Below is an example of organizing a communication using all three of LOINC, SNOMED, and a custom code.
+
+```
+{
+  resourceType: "Communication",
+	//...
+	category: [
+	  // A category coded using LOINC
+		{
+			text: "Practitioner",
+			coding: [
+				{
+					code: "18601-5"
+					system: "http://loinc.org",
+          display: "Primary practitioner profession"
+				}
+			]
+		},
+	],
+	//...
+	topic: {
+	  // A topic coded using SNOMED
+		text: "High blood pressure result for John Doe, from test performed 8/2/2023",
+		coding: [
+			{
+				code: "38341003",
+				system: "http://snomed.info/sct",
+				display: "Hypertensive Disorder"
+			}
+		]
+	}
+}
+
+{
+	resourceType: "Communication",
+	//...
+	// An example of how an internal custom coding could be used for both category and topic
+	category: [
+		{
+			text: "appointment-reminder",
+			coding: [
+				{
+					code: "apt-rem",
+					system: "https://example-practice.org"
+				}
+			]
+		}
+	],
+	//...
+	topic: {
+		text: "annual-physical"
+		coding: [
+			{
+				code: "annu-phys",
+				system: "https://example-practice.org"
+			}
+		]
+	}
+}
+```
+
+## Search and GraphQL
+
+Effectively grouping and filtering your communications makes it easier to search and query for specific data.
+
+For example, if you want to get all of the communications between the provider and a specific patient, you could use the following query. In this, we query for a patient by their id, then search all of the communications that reference them. From these communications, we get the text of the communication, the sender and time it was sent, and the receiver and time it was received. Preview this query in [GraphiQL](graphiql.medplum.com)
+
+```
+{
+	# Search for a specific patient by id
+  Patient(id: "example-patient-id") {
```

Comment:
Our graphQL examples typically don't use fragments. Try to follow the examples here: 

https://www.medplum.com/docs/graphql/basic-queries#resolving-nested-resources-with-the-resource-element